src/rpc/blockchain.h

```diff
@@ -48,11 +48,18 @@ UniValue blockheaderToJSON(const CBlockIndex* tip, const CBlockIndex* blockindex
 /** Used by getblockstats to get feerates at different percentiles by weight  */
 void CalculatePercentilesByWeight(CAmount result[NUM_GETBLOCKSTATS_PERCENTILES], std::vector<std::pair<CAmount, int64_t>>& scores, int64_t total_weight);
 
+/** File format type for CreateUTXOSnapshot() */
+enum class UTXOSnapshotFormat {
+    COMPACT,
```

Comment:
nit: is there a reason to call the original format `COMPACT` ?
Something like `BINARY` might be more accurate.

```suggestion
    BINARY,
```

Reply:
No fundamental reason. I just used nomenclature from the CSV attempt: #18689.
Happy to change if we think it's clearer and people have strong opinions about it.

Reply:
I like compact more than binary.